lib/cocoapods/target/framework_paths.rb

```diff
@@ -0,0 +1,30 @@
+module Pod
+  class FrameworkPaths
+    # @return [String] the path to the .framework
+    #
+    attr_reader :source_path
+
+    # @return [String, Nil] the dSYM path, if one exists
+    #
+    attr_reader :dsym_path
+
+    def initialize(source_path, dsym_path)
```

Comment:
nit can make `dsym_path = nil` as default

Reply:
this way you can get rid of the `, nil` everywhere else

Reply:
yeah makes sense